### PR TITLE
Adding Quick-Start Guide to docs

### PR DESCRIPTION
**Pull request by ramonpzg**
This Quick-Start guide provides new users with a straightforward set of instructions on how to get up and running with MLServer. In addition, it provides seasoned users with a fresh perspective on how to explain some of the features of MLServer.

The Quick-Start guide is divided into the following sections.
- 00 What is MLServer?
- 01 Dependencies
- 02 Set Up
- 03 Building a Service
- 04 Testing our Service
- 05 Creating Model Replicas
- 06 Packaging our Service

There are also a few images being added to the assets directory inside the docs.

There is one more minor change to the main docs `conf.py` file. Instead of saying "2021, Seldon Technologies" it now says "**2023**, Seldon Technologies".

### CODE REVIEW

File: docs/quick-start/index.md [hunk 1]
```diff
@@ -0,0 +1,487 @@
+# MLServer Quick-Start Guide
```
Comment (edshee) [inline docs/quick-start/index.md:1]: I still think calling it a quick start guide is misleading. It's a tutorial that walks through a specific learning path to achieve a use-case. (See https://diataxis.fr/ for more on definitions).

In contrast, a "quickstart" is something that is designed to get you up and running with the **basics** of the tool in just a few minutes.

File: docs/quick-start/index.md [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+
+MLServer is an open-source Python library for building production-ready asynchronous APIs for machine learning models.
+
+## 01 Dependencies
+
+The first step is to install `mlserver`, the `spacy` library, and the [language model](https://spacy.io/models/en#en_core_web_lg) 
+`spacy` will need for our use case. We will also download the `wikipedia-api` library to 
+test our use case with a few fun summaries.
+
+If you've never heard of [spaCy](https://spacy.io/) before, it is an open-source Python
```
Comment (edshee) [inline docs/quick-start/index.md:20]: :+1:

File: docs/quick-start/index.md [hunk 3]
```diff
@@ -0,0 +1,487 @@
...
+
+
+```python
+mkdir -p models_hub/similarity_model
+```
+
+Before we create a service that allows us to compare the similarity between two documents, it is 
+good practice to first test that our solution works first, especially if we're using a pre-trained 
+model and/or a pipeline.
+
```
Comment (edshee) [inline docs/quick-start/index.md:62]: Maybe tell them to make a python file here? e.g. `app.py`

File: docs/quick-start/index.md [hunk 4]
```diff
@@ -0,0 +1,487 @@
...
+```python
+barbie = wiki_wiki.page('Barbie_(film)').summary
+oppenheimer = wiki_wiki.page('Oppenheimer_(film)').summary
+
+print(barbie)
+print()
+print(oppenheimer)
+```
+
```
Comment (edshee) [inline docs/quick-start/index.md:103]: and then tell them to run it here? `python app.py`

File: docs/quick-start/index.md [hunk 5]
```diff
@@ -0,0 +1,487 @@
...
+At its core, MLServer requires that users give it 3 things, a `model-settings.json` file with 
+information about the model, an (optional) `settings.json` file with information related to the server you 
+are about to set up, and a `.py` file with the load-predict recipe for your model (as shown in the 
+picture above).
+
+Let's create a directory for our model.
+
+
+```python
+mkdir -p models_hub/similarity_model
```
Comment (edshee) [inline docs/quick-start/index.md:56]: Slightly nitpicking here but is there any reason we need to create a directory? They could just create the files all in a flat folder structure and then you don't need to append `models_hub/similarity_model` to everything?

File: docs/quick-start/index.md [hunk 6]
```diff
@@ -0,0 +1,487 @@
...
+to our service and use the `requests` library to [POST](https://www.baeldung.com/cs/http-get-vs-post) our request.
+
+
+```python
+from mlserver.codecs import StringCodec
+import requests
+```
+
+Please note that the request below uses the variables we created earlier with the summaries 
+of Barbie and Oppenheimer. If you are sending this POST request from a fresh python file, make
```
Comment (edshee) [inline docs/quick-start/index.md:235]: :+1:

File: docs/quick-start/index.md [hunk 7]
```diff
@@ -0,0 +1,487 @@
...
+        StringCodec.encode_input(name='docs', payload=[barbie, oppenheimer], use_bytes=False).dict()
+    ]
+}
+inference_request
+```
+
+
+
+
+    {'inputs': [{'name': 'docs',
```
Comment (edshee) [inline docs/quick-start/index.md:251]: You seem to have included the inference request twice? Once with variables and once with the raw text. I'd suggest just keeping the one with variables as you have already explained above that they'll need to be able to reference them from earlier

**Review by edshee — CHANGES_REQUESTED** [review]
A few changes on the readme but otherwise a very good piece of content. Great work Ramon. I particularly love the images!

File: docs/quick-start/index.md (hunk 1 above)
Comment (ramonpzg) [reply docs/quick-start/index.md:1]: Fair point. How about calling this section "Getting Started"?

File: docs/quick-start/index.md (hunk 5 above)
Comment (ramonpzg) [reply docs/quick-start/index.md:56]: When I started writing this I intended to create multiple services so the `models_hub` dir made sense. Not needed now so I'll remove it

File: docs/index.md [hunk 8]
```diff
@@ -1,6 +1,7 @@
 ```{toctree}
 :hidden:
 :titlesonly:
+Getting Started <./getting-started/index>
```
Comment (edshee) [inline docs/index.md:4]: Probably better to have this under the "Examples" section rather than it's own heading. It could go under the "tutorials" bit.

**Review by edshee — CHANGES_REQUESTED** [review]

**Review by edshee — APPROVED** [review]

**Pull request merged**